templates/rabbitmq.config.erb

```diff
@@ -5,9 +5,12 @@
   {ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]},
 <%- end -%>
   {rabbit, [
-<% if @ldap_auth -%>
+<% if @ldap_auth and (@auth_backends == 'UNSET') -%>
     {auth_backends, [rabbit_auth_backend_internal, rabbit_auth_backend_ldap]},
 <% end -%>
+<% if @auth_backends != 'UNSET' -%>
+    {auth_backends, <%= @auth_backends %>},
```

Comment:
Hi David,
Thank you for re-considering my request :)I re-submitted using the same setup as ssl_versions, I took the liberty of cleaning up my logic in the template a bit more and adding in another test.Let me know if there's anything else you'd like done with it,
Megan (firefly4268)
 

      From: David Schmitt <notifications@github.com>
 To: puppetlabs/puppetlabs-rabbitmq <puppetlabs-rabbitmq@noreply.github.com> 
Cc: firefly4268 <megan.miller@ymail.com>
 Sent: Thursday, January 14, 2016 9:39 AM
 Subject: Re: [puppetlabs-rabbitmq] Module2940 addition auth backends variable (#424)
   
In templates/rabbitmq.config.erb:>      {auth_backends, [rabbit_auth_backend_internal, rabbit_auth_backend_ldap]},
>  <% end -%>
> +<% if @auth_backends != 'UNSET' -%>
> +    {auth_backends, <%= @auth_backends %>},
I'd prefer that to be handled in the same way as ssl_versions (See line 5 above). Tthe user passes in an array and the template expands this into a properly formatted configuration entry.—
Reply to this email directly or view it on GitHub.